File: sdkwork-app-sdk-go/types/generation_task_vo.go
package types

// 生成任务响应
type GenerationTaskVO struct {
	CreatedAt string `json:"createdAt"`
	UpdatedAt string `json:"updatedAt"`
	TaskId int `json:"taskId"`
	RequestId string `json:"requestId"`
	Type string `json:"type"`
	Model string `json:"model"`
	Channel string `json:"channel"`
	Status string `json:"status"`
	Progress int `json:"progress"`
	InputParams GenerationInput `json:"inputParams"`
	OutputResult GenerationOutput `json:"outputResult"`
	Cost float64 `json:"cost"`
	ErrorCode string `json:"errorCode"`
	ErrorMessage string `json:"errorMessage"`
	RetryCount int `json:"retryCount"`
	MaxRetry int `json:"maxRetry"`
	StartedAt string `json:"startedAt"`
	CompletedAt string `json:"completedAt"`
	ConversationId int `json:"conversationId"`
	MessageId int `json:"messageId"`
	ParentId int `json:"parentId"`
	BatchId string `json:"batchId"`
	BizScene string `json:"bizScene"`
	BizId int `json:"bizId"`
	Done bool `json:"done"`
}

File: sdkwork-app-sdk-python/sdkwork_app_sdk/models/generation_task_vo.py
from dataclasses import dataclass
from typing import Optional, List, Dict, Any

@dataclass
class GenerationTaskVO:
    """生成任务响应"""
    created_at: str = None
    updated_at: str = None
    task_id: int = None
    request_id: str = None
    type: str = None
    model: str = None
    channel: str = None
    status: str = None
    progress: int = None
    input_params: GenerationInput = None
    output_result: GenerationOutput = None
    cost: float = None
    error_code: str = None
    error_message: str = None
    retry_count: int = None
    max_retry: int = None
    started_at: str = None
    completed_at: str = None
    conversation_id: int = None
    message_id: int = None
    parent_id: int = None
    batch_id: str = None
    biz_scene: str = None
    biz_id: int = None
    done: bool = None
